sky/services/platform/filesystem.mojom

```diff
@@ -0,0 +1,38 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+[DartPackage="sky_services"]
+module filesystem;
+
+/// Returns commonly used locations on the filesystem
+interface Filesystem {
+  /// Path to the read-only directory that contains the assets packaged with
+  /// the application bundle.
+  ///
+  /// Example(s):
+  ///    iOS: The application bundle directory
+  AssetsDirectory() => (string path /* non-null */);
```

Comment:
why `/* non-null */` ? I thought you had to be explicit about nullability in mojom.